e2etest/bgptests/validate.go

```diff
@@ -193,6 +193,22 @@ func checkServiceOnlyOnNodes(cs clientset.Interface, svc *corev1.Service, expect
 	}
 }
 
+func checkServiceNotOnNodes(cs clientset.Interface, svc *corev1.Service, expectedNodes []corev1.Node, ipFamily ipfamily.Family) {
+	if len(expectedNodes) == 0 {
+		return
+	}
+	ip := svc.Status.LoadBalancer.Ingress[0].IP
+	nodeIps := k8s.NodeIPsForFamily(expectedNodes, ipFamily)
+
+	for _, c := range FRRContainers {
+		Eventually(func() bool {
+			routes, err := frr.RoutesForFamily(c, ipFamily)
+			framework.ExpectNoError(err)
+			return len(routes[ip].NextHops) == len(nodeIps)
```

Comment:
sorry just noticed this. This is a bit fragile, as we are checking the lenght and a double bug may result in having them being different but with the svc being there.
Can we check that the nodes are really not there?